maas-region/src/charm.py

```diff
@@ -179,15 +181,16 @@ def enrollment_token(self) -> Union[str, None]:
         return MaasHelper.get_maas_secret()
 
     @property
-    def bind_address(self) -> Union[str, None]:
+    def bind_address(self) -> str:
```

Comment:
If we're no longer going to return None here, then the walrus check for `bind_address` in `maas_api_url` no longer makes sense. Shouldn't change the behavior if we remove it, however.